include/Graphs/VFGNode.h

```diff
@@ -88,6 +88,12 @@ class VFGNode : public GenericVFGNodeTy
         return icfgNode->getFun();
     }
 
+    /// Return the corresponding LLVM value, if possible, nullptr otherwise.
+    virtual const llvm::Value* getLLVMValue() const
```

Comment:
We can remove the llvm:: name space here. Also, we will rewrite LLVM Value later. Could just change ``getLLVMValue'' to ``getValue``?

Reply:
Will do. What do mean with "we will rewrite LLVM Value later"?
Namespace dropping is possible because `llvm::Value` is typedefed to `SVF::Value`, isn't it (generally, dropping namespaces in headers is not desirable)?

Reply:
Later, we may use `Value` as an interface, it could be an LLVM Value wrapper and could be something else for other compilers.